app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/points_stages/DrawCornerHistoryStage.kt

```diff
@@ -0,0 +1,166 @@
+package dk.scuffed.whiteboardapp.pipeline.stages.points_stages
```

Comment:
Why are we creating a whole new stage here? Can't we just reuse drawpointsstage?